src/styles/index.scss

```diff
@@ -1,3 +1,82 @@
-body {
+$main-accent: #060b35;
+$secondary: #616070;
+$blue_accent: #00acdc;
+
+@import "blocks/card";
+@import "blocks/stars";
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+html {
+  background-color: darkgray;
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+.block__reviews {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  font-size: 10px;
+  line-height: 14px;
+  color: $main-accent;
+  text-align: right;
```

Comment:
remove :
  display: flex;
  flex-direction: column;
  justify-content: flex-end;
  text-align: right;
  
  Redundant styles,  you have to add display: flex if you have block with at least two elements